example/helm/values-with-pyroscope.yaml

```
@@ -0,0 +1,79 @@
+defaults:
+  image:
+    tag: v1.4
+  time_zone: Asia/Tokyo
+  logging:
+    level: debug
+  server_config:
+    metrics:
+      pprof:
+        enabled: true
```

I think it is better to let readers know which config enables Pyroscope, so I suggest:
```suggestion
        enabled: true # enable Pyroscope
```